### ISSUE TITLE
missing associated changeset ID on resolved note and hyperlink from changeset to note

### ISSUE THREAD

**Issue by brierjon** [openstreetmap/iD#9510]
### URL

https://www.openstreetmap.org/changeset/132989455

### How to reproduce the issue?

1. Create an OSM note to address in a changeset.
2. Make edits in iD Editor to start a changeset
3. Close the note ([https://www.openstreetmap.org/note/3573370](https://www.openstreetmap.org/note/3573370))
4. Click on the changeset ([https://www.openstreetmap.org/changeset/132989455](https://www.openstreetmap.org/changeset/132989455))
5. See that the value for the closed:Note numbers do not link back to OSM note to review contents of the note.

Part 1 - Missing hyperlink - The value for key closed:note should be a hyperlink in the format  https://www.openstreetmap.org/note/(note ID value here)

Part 2. - Missing reverse linking from Note to Changeset
In the example changeset and note above, navigation is unidirectional from the changeset to the note, but not from the note back to the changeset where the note was closed. Its total guesswork of if a closed note was fixed in one or more changsets without sifting through the history.

While the notes may disappear from the map, these are not gone from the database. It would be helpful for anyone viewing or reviewing old notes to find associated changeset(s) where it is possible to include a bidirectional link between changset(s) and note(s). Notes currently do not include (at least in the UI) which changeset is associated when they are closed. In this example the note was closed and associated in the changeset, but the note does not include the changeset ID. No changeset ID is hyperlinked to view the associated changeset that the note was resolved from.

Possible additional conditions to address:
1. partially resolved notes in a changeset (rather than binary closed or not closed, some notes have more than one part and are some parts are simple to address while other parts might need much more extensive work or different imagery to address) - not as common, but does affect linking more than one changeset to a note when it does occur.

### Screenshot(s) or anything else?

![image](https://user-images.githubusercontent.com/717866/221324403-f64322cb-7e73-4c19-aa49-193ad66ca2bc.png)
![image](https://user-images.githubusercontent.com/717866/221324537-df884f04-4c9d-4894-a9a2-4ee12c5bcf49.png)

Other than my own comments adding the note ID in the changeset comment, locating a changeset associated with a note in the UI is nearly impossible.
![image](https://user-images.githubusercontent.com/717866/221325585-7e278ae8-89e0-4b32-bafe-ed7743ad9065.png)

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.24.2

### Which browsers are you seeing this problem on?

Chrome, Firefox, Microsoft Edge, Opera, Safari, Vivaldi

**Comment by danieldegroot2** [comment]
Hey brierjon,

In the meantime, have a look at the [noteSolver](https://wiki.openstreetmap.org/wiki/User:Kmpoppe/Plugins#noteSolver) JOSM plugin if you haven't yet.

cc: @kmpoppe

**Comment by k-yle** [comment]
Part 1 of this issue will be solved by https://github.com/openstreetmap/openstreetmap-website/pull/2621 Both parts of this issue are not related to iD so you might want to transfer this issue to https://github.com/openstreetmap/openstreetmap-website

**Comment by gy-mate** [comment]
> you might want to transfer this issue to https://github.com/openstreetmap/openstreetmap-website

I agree. @k-yle, as a collaborator, do you have the rights to transfer this issue there?

**Comment by k-yle** [comment]
I can't transfer it sorry, you need to be a maintainer of both repositories to transfer an issue

**Comment by gy-mate** [comment]
Was _part 2_ of this issue also resolved by https://github.com/openstreetmap/openstreetmap-website/pull/6197?

**Comment by k-yle** [comment]
well, no, but both parts are effectively https://github.com/openstreetmap/iD/labels/wontfix-not-iD so i think it's fair to close this issue.

For completeness:
 - part 1 was solved by [openstreetmap-website#6197](https://github.com/openstreetmap/openstreetmap-website/pull/6197)
 - part 2 is solvable if & when [openstreetmap-website#5294](https://github.com/openstreetmap/openstreetmap-website/issues/5294) is implemented

**Comment by gy-mate** [comment]
I see—thanks for the links!
